fpm/test/fpm_test/main.f90

```diff
@@ -18,8 +17,7 @@ program fpm_testing
     testsuite = [ &
         & new_testsuite("fpm_toml", collect_toml), &
         & new_testsuite("fpm_manifest", collect_manifest), &
-        & new_testsuite("fpm_source_parsing", collect_source_parsing), &
-        & new_testsuite("fpm_versioning", collect_versioning) &
+        & new_testsuite("fpm_source_parsing", collect_source_parsing) &
```

Comment:
Is there a particular reason to remove the versioning tests?

Reply:
Did not directly change anything in fpm/test/fpm_test/main.f90 intentionally, but I saw errors when I pushed the last set of changes; and have seen them with this PR.

Reply:
No. that file was not directly changed by editing. When I revert that change the test does not run though. There were errors when I ran git but they did not indicate that file was involved.

Reply:
I missed this. Can you add the versioning test back in along with the use statement:
https://github.com/fortran-lang/fpm/blob/0c35749e0d90b5de43a7a90eb47677695e5c81e2/fpm/test/fpm_test/main.f90#L9

and we can sort out any regression. (I've tried this locally and I don't get any errors)
This may simply be the result of a bad merge.